src/CTA.Rules.PortCore/SolutionPort.cs

```diff
@@ -477,6 +477,22 @@ internal ProjectType GetProjectType(FeatureDetectionResult projectTypeFeatureRes
             {
                 return ProjectType.WCFClient;
             }
+            else if(projectTypeFeatureResult.IsVBNetMvcProject())
+            {
+                return ProjectType.VBNetMvc;
+            }
+            else if (projectTypeFeatureResult.IsVBWebFormsProject())
```

Comment:
I could be misreading the logic, but won't this never get hit since the condition for non-VB projects doesn't check for C# or some other language. So the AspNetWebFormsFeature will return true before it gets to check if it's also VB specific

Reply:
If this is actually an issue, it would be worth checking all the other VB types. Also, is there a test that covers this case?

Reply:
that is a good point actually, will change the ordering